api/interservice/infra_proxy/request/nodes.proto

```diff
@@ -24,3 +26,28 @@ message DeleteNode {
     // Node name.
     string name = 3;
 }
+
+message UpdateNode {
+    // Chef organization ID.
+    string org_id = 1;
+    // Chef Infra Server ID.
+    string server_id = 2;
+    // Node name.
+    string name = 3;
+    // Node environment.
+    string environment = 4;
+    // Node policy name.
+    string policy_name = 5;
+    // Node policy group.
+    string policy_group = 6;
+    // Node run-list.
+    repeated string run_list = 7;
+    // Node automatic JSON attributes.
+    google.protobuf.Struct automatic_attributes = 8;
+    // Node default JSON attributes.
+    google.protobuf.Struct default_attributes = 9;
+    // Node normal JSON attributes.
+    google.protobuf.Struct normal_attributes = 10;
+    // Node override JSON attributes.
+    google.protobuf.Struct override_attributes = 11;
```

Comment:
Need to make these consistent. The previous uses are written like this:
```suggestion
    // Node override attributes JSON.
    google.protobuf.Struct override_attributes = 11;
```
That is "attributes JSON", not "JSON attributes". This applies to all four variations: automatic_attributes, default_attributes, normal_attributes, and override_attributes.

Also, please review all other existing uses for consistency. Searching for each of those four attributes in other proto files reveals that, in api/interservice/infra_proxy/response/policyfiles.proto, override_attributes says "Stringified JSON of the override attributes". Let's standardize by changing that to "Policyfile override attributes JSON." Similarly in these files:
api/external/infra_proxy/response/roles.proto
api/interservice/infra_proxy/response/environments.proto
api/interservice/infra_proxy/response/policyfiles.proto
api/interservice/infra_proxy/response/roles.proto